electron/index.js

```diff
@@ -153,63 +154,147 @@ ipcMain.on("exit-app", (event, flag) => {
 });
 
 function performFsJob(job) {
-    const fname = path.join(storePath, job.filename);
-
-    switch (job.type) {
-        case "read": {
-            if (!fs.existsSync(fname)) {
-                return {
-                    // Special FILE_NOT_FOUND error code
-                    error: "file_not_found",
-                };
-            }
-
-            let contents = "";
-            try {
-                contents = fs.readFileSync(fname, { encoding: "utf8" });
-            } catch (ex) {
-                return {
-                    error: ex,
-                };
-            }
-
-            return {
-                success: true,
-                data: contents,
-            };
-        }
-        case "write": {
-            try {
-                fs.writeFileSync(fname, job.contents);
-            } catch (ex) {
-                return {
-                    error: ex,
-                };
-            }
-
-            return {
-                success: true,
-                data: job.contents,
-            };
+    if (job.mods) {
```

Comment:
I think it is unnecessary here to make it an if-else or? instead just compute the fname differently?

Reply:
also, maybe we can somehow ensure that the game can not read any file outside of its game folder? to make sure mods can not access sensitive data

Reply:
I've done this now

Reply:
> maybe we can somehow ensure that the game can not read any file outside of its game folder?